src/utilities/new-sequence-editor/sequence-completion.ts

```diff
@@ -189,7 +191,7 @@ export function sequenceCompletion(
       //   }
       // }
 
-      const globals = globalThis.GLOBALS;
+      const globals = get(sequenceAdaptation)?.globals ?? [];
```

Comment:
Can the store define the methods that perform this check so that multiple files don't have to?